client/src/components/events-components/EventList.tsx

```diff
@@ -1,7 +1,7 @@
 import EventCard from "./EventCard";
 
 const EventList = () => {
-  const events = [{}, {}];
+  const events = [{ name: "john" }, { name: "Hernyr" }];
   return (
     <section className="flex justify-start">
```

Comment:
curious why do we use `section` here? this HTML element usually represents a section in the document:
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/section

for a list, the convention is to use `ul/ol`:
https://www.w3schools.com/tags/tag_ul.asp